Exercises/NoModules/Chapter 12/Soln12_03/Integer.h

```diff
@@ -0,0 +1,25 @@
+#ifndef INTEGER_H
+#define INTEGER_H
+
+class Integer
+{
+public:
+  Integer(int value = 0);
```

Comment:
Isn't the solution with a defaulted default constructor preferable?

Reply:
Same remark for subsequent solutions. You choose.

Reply:
I don't know of any objective reason to prefer initializing this member to the default constructor argument here, so I'll leave it.